manage.py

```diff
@@ -15,11 +16,20 @@ class Development(LocalRoot):
 @Development.register
 @local('remainder', metavar='alembic arguments', nargs=argparse.REMAINDER)
 def alembic(context, args):
-    """Configuration wrapper to use the Alembic schema migrations library for Triage development"""
+    """Configuration wrapper to use the Alembic schema migrations library for Triage development.
+    Try alembic -h to see description of all available subcommands"""
```

Comment:
Not a bad note to include; but, might be at least as useful to explain that these should also work:

```
manage alembic - -h
manage alembic -- -h
```

It's an annoying thing to get around `argparse` seeing the help flag and immediately exiting, (but also not uncommon with other CLIs, even Git's).

Reply:
Ohhhh I was wondering if there was a way around that